exercises/Robert_Applin/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,203 @@
+/**
+ * Skeleton main routine
+ */
+
+#include <algorithm>
+#include <cctype>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <iterator>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+using WordCount = std::vector<std::pair<std::string, std::size_t>>;
```

Comment:
Good. Using a type alias here makes a lot of sense. 👍

Reply:
I would consider perhaps even type aliasing the `std::pair` as well as this is used in several places.